---
layout: page
title: My Research Endeavours!
permalink: /research/
---

{: style="text-align: left"}
### Meta Reinforcement Learning - Reward Decompostion

I'm working with Jack Lindsey as part of the Litwin-Kumar Lab at the Zuckerman Institute on developing meta-RL algorithms that learn more efficient learning rules. I'm further interested in RL algorithms that decompose the reward signal to accelerate learning.

{: style="text-align: left"}
### Generalized Multi-Agent Framework To Simulate Deviant Mining Strategies on Bitcoin-like Blockchains

We first present a taxonomy of existing deviant mining strategies and parametrise them using a common set of variables. We then design a simulator that is capable of testing proof-of-work blockchain systems for this repertoire in a _multi-agent_ setting, and provide insight. Check out our poster [here]({{ site.baseurl }}/documents/blockchain.pdf), a paper draft is in the works! Work done with the Math Department at Columbia University

{: style="text-align: left"}
### Interactive Metric Learning under sparse signaling

This is an incomplete project (currently paused). I'm working on developing a metric learning algorithm that performs perceptron-like online updates using split-merge signals on data clusters. Advised by [Prof. Nakul Verma](http://www.cs.columbia.edu/~verma/)

{: style="text-align: left"}
### Memorial Sloan Kettering Cancer Centre - [CBSP Program](https://www.mskcc.org/education-training/high-school-college/computational-biology-summer-program-alumni)

I got paired with MSKCC's [m-ski Lab](https://www.mskilab.org/#intro) as a part of the CBSP summer program. Unfortunately this experience was cut short due to Covid. I replicated analyses with [dryclean](https://github.com/mskilab/dryclean) - a statistical tool to denoise genomic coverage data. 

{: style="text-align: left"}
### Predicting gene expression using Transfer Learning and Attention

With [Shiu Lab](https://shiulab.github.io/people_past/) at Michigan State University, I built an interpretable deep learning model equipped with the attention mechanism that predicted whether genes in certain crops were expressed in response to environmental stress. Futhermore we transfer learnt TF binding motifs from A. thaliana to O. sativa and showed how certain binding sites were preserved across species. See the [poster]({{ site.baseurl }}/documents/ACRES_Poster.pdf) that I presented at the MidSure research symposium

{: style="text-align: left"}
### 2017: SSP Biochemistry - Designing a fungal enzyme inhibitor
I worked on a mentored research project with a team, as part of [SSP Biochemistry](https://summerscience.org/the-ssp-experience/fungal-inhibitor-design/) at Purdue University, to design a fungal enzyme inhibitor using wet lab techniques as well as computational modelling and simulation software.

Publication:  Duong, L., Dunakey, S., Jog, K., Sriram, A., Tian, M., Wang, A., Hall, M. C. (2020). [Characterization of the Cdc14 phosphatase homolog from Ustilago maydis](https://doi.org/10.4231/RZTW-V362). Purdue University Research Repository. [doi:10.4231/RZTW-V362](https://doi.org/10.4231/RZTW-V362)
